.travis.yml

```diff
@@ -0,0 +1,20 @@
+language: objective-c
+osx_image: xcode7.2
+branches:
+  only:
+  - "/^refactor\\/.*$/"
+  - "/^feature\\/.*$/"
+  - "/^bug\\/.*$/"
+  - "/^test\\/.*$/"
+  - "/^chore\\/.*$/"
+  - "/^release.*$/"
+notifications:
+  email: false
+before_install:
+- brew update || brew update
```

Comment:
To be fair, I really don't know. I put it here because we had 2 regressions with brew, and I prefer to do it to avoid being blocked